viewer/src/app/feature-service.service.ts

```diff
@@ -96,9 +98,42 @@ export type featureCollectionGeoJSON = {
   numberReturned?: number
 }
 
+export type DataProjectionMapping = {
+  dataProjection: ProjectionLike //Projection of the data we are reading
+  visualProjection: ProjectionLike //Projection of the feature geometries created by this function
+}
+
 export type DataUrl = {
   url: string
-  projection: ProjectionLike
+  dataMapping: DataProjectionMapping
+}
+export const defaultMapping: DataProjectionMapping = { dataProjection: 'EPSG:4326', visualProjection: 'EPSG:3857' }
+
+export function projectionAttribute(value: ProjectionLike = 'HTTP://WWW.OPENGIS.NET/DEF/CRS/OGC/1.3/CRS84'): DataProjectionMapping {
```

Comment:
Waarom de URL in hoofdletters?

Snap de term 'attribute' niet helemaal. Misschien `getProjectionMapping()` noemen?

Reply:
naamgeving was in lijn met b.v. https://angular.io/api/core/booleanAttribute, maar omdat dit in een getter gebruikt en iets meer conversie in zit is jouw voorstel beter.

Reply:
Waarom de URL in hoofdletters? Omdat het kan 😄 => aangepast